src/components/views/right_panel/RoomSummaryCard.tsx

```diff
@@ -150,9 +150,15 @@ const AppRow: React.FC<IAppRowProps> = ({ app, room }) => {
             className="mx_RoomSummaryCard_icon_app"
             onClick={onOpenWidgetClick}
             // only show a tooltip if the widget is pinned
-            title={isPinned ? _t("Unpin a widget to view it in this panel") : ""}
-            forceHide={!isPinned}
-            disabled={isPinned}
+            title={
+                isPinned
+                    ? _t("Unpin this widget to view it in this panel")
+                    : isMaximised
+                        ? _t("Close this widget to view it in this panel") :
+                        ""
```

Comment:
Let's extract this logic to regular `if ... else` blocks above, similar to `pinTitle`. Nested ternaries like this are usually a sign that a different style would be better.

Reply:
yea its a little bit weired. I agree.